docs/essentials/building-monorepo-projects.md

```diff
@@ -241,6 +241,24 @@ blocks:
       when: "change_in('../Gemfile.lock')"
 ```
 
+### When any Javascript file changes
+
+```yaml
+blocks:
+  - name: JS tests
+    run:
+      when: "change_in('/assets/**/*.js')
+```
+
+### When any file changes, except docs
+
+```yaml
+blocks:
+  - name: Test Web Server
+    run:
+      when: "change_in('/web-app/', exclude: ['/web-app/docs'])"
```

Comment:
The `exclude` option needs to be within the configuration map, e.g.
`change_in('/web-app/', {exclude: ['/web-app/docs']})`